external/feature.md

```diff
@@ -0,0 +1,9 @@
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Purus in mollis nunc sed. Diam ut venenatis tellus in metus vulputate. Tristique et egestas quis ipsum suspendisse ultrices. Platea dictumst vestibulum rhoncus est pellentesque. Eu non diam phasellus vestibulum lorem sed risus. Semper risus in hendrerit gravida. Pharetra massa massa ultricies mi quis hendrerit dolor magna. Urna id volutpat lacus laoreet. Lacus suspendisse faucibus interdum posuere lorem. Mauris pharetra et ultrices neque ornare. Sed lectus vestibulum mattis ullamcorper velit sed. Non odio euismod lacinia at. In mollis nunc sed id. Molestie ac feugiat sed lectus vestibulum mattis ullamcorper velit. Gravida cum sociis natoque penatibus et magnis. Tincidunt dui ut ornare lectus. Integer quis auctor elit sed.
```

Comment:
comment on first paragraph

Reply:
second comment on the first para.

This is a multi line comment.

Reply:
another comment

Reply:
checking more

Reply:
some more comments but this time it is multi line
line 2
line 3
line 4

line6